include/onnxruntime/core/graph/graph.h

```diff
@@ -1179,6 +1215,9 @@ class Graph {
   // This pointer is owned by parent model.
   ONNX_NAMESPACE::GraphProto* graph_proto_;
 
+  // GraphProto that provides storage for the ONNX proto types deserialized from a flexbuffer/flatbuffer
+  ONNX_NAMESPACE::GraphProto deserialized_proto_data_;
+
```

Comment:
Note: This is for convenience short term. In the 1.6 timeframe it would be better to go straight to OrtValue from the flatbuffer (can refer to offset in buffer and avoid copy) instead of using TensorProto. TensorProto is copied to OrtValue when SessionState is finalized.